contexts/lds-jws2020-v1.jsonld

```diff
@@ -0,0 +1,53 @@
+{
```

Comment:
Same note about tracking formalizing signing schema into spec.

Reply:
Should be covered by https://github.com/spruceid/ssi/issues/116? Each proof type has its corresponding verification method that should be specified. e.g. how https://github.com/w3c-ccg/lds-jws2020 defines [JsonWebSignature2020](https://w3c-ccg.github.io/lds-jws2020/#dfn-jsonwebsignature2020) and [JsonWebKey2020](https://w3c-ccg.github.io/lds-jws2020/#dfn-jsonwebkey2020).